### ISSUE TITLE
Flaws, the Negitive Knack

### ISSUE THREAD

**Issue by CaspianWild** [Arx-Game/arxcode#205]
There was talk in the event channel about negative knacks so I decided to propose hear. These would work much like a knack, but in the opposite direction. Every level of a Flaw gives you -2 to a stat/skill combo.

Flaws could cost XP, but very little exp, maybe a flat 1-5 xp per level. Or picking a flaw could give you 5 xp per level, to a max of level 5.

I realize that check/flub is a thing, but some people want it to be random if they succeed or fail.

**Comment by dvoraen** [comment]
So, I personally am not against the idea in and of itself, but my major question is: "Why would you want to pay for a detriment?"  From a design standpoint, in terms of acquisition, I don't like it.  From a "play to your strengths, demonstrate your weaknesses" standpoint, I am absolutely on board.  My PC, for example, will never be a social creature.  The closest he'd come to modeling clothes is showing up in them.

In fact, I could easily see him getting a flaw about certain social graces (a flaw for Charm+Etiquette, perhaps), **BUT** ==> _I would absolutely hate to spend XP to get it_, as a player.  I'd rather the handicap come with an incentive for having it.  Whether that's XP or higher AP regen (or AP cap?) I don't know.  I think flaws would have to be capped on principle though (like -5 or -10 to the roll, or scaling based on difficulty, or -1 die, etc.).  At the minimum, I think they should be a "have it or don't have it" detriment rather than "how much of a clutz are you?"

**Comment by rozzingit** [comment]
Yeah, I feel like -- in RPG systems that do allow characters to basically mark certain things as flaws, they usually get something back out of it. The idea of _paying_ to mark my character as bad at something seems strange.

**Comment by dvoraen** [comment]
Kind of piggybacking on my previous thoughts, but I _can_ see XP being used to negate or mitigate a flaw on a temporary basis.  I'm perfectly fine with that idea of an XP sink.

**Comment by CaspianWild** [comment]
I'll admit spending xp was not the ideal choice, but I couldn't figure out a way to not have people pick weird Flaws just for the benefit knowing they will rarely ever roll those flaws. Though I do like the idea of getting Bonus AP for having a flaw. Though getting bonus AP for having a flaw is still pretty strong unless it's getting 1 AP per level and still have to pay XP for it.

**Comment by TehomCD** [comment]
I like the idea though obviously the utility of knacks/flaws vary wildly based on skill/stat combinations due to the probability of some never, ever being used. Due to that, we'd want to gate people from gaining mechanical benefit for selecting flaws that could never impede them in any way.

One thing that's on the roadmap is to eventually move to having checks be formalized, albeit crowdsourced: ie, people come to a consensus about stat/skill/difficulty combinations in specific situations, and thereafter that's what's used. Forcing flaws to conform to specific checks would probably address that concern.

**Comment by CaspianWild** [comment]
I could see two ways of handling that:
Have flaws be requested for approval
Have the players write out common flaws and submit them for approval, and that list can be the list of flaws you can take.

**Comment by Selerik** [comment]
While I like the idea, the only way to properly ensure flaws come up is to
in some way rewards players only after their flaw impacts play. Which,
sadly, requires admin time to review and approve.

My leaning here is to add flaws at no xp cost/reward for players that want
to dabble in them, but defer adding any additional support to flaws until
after existing systems and the workloads they're generating are stable.
Encourage players to tie them to their goals, and leave it at that.

**Comment by dreamsofgold** [comment]
I don't see the worth of adding in mechanical flaws when historically the community has dealt with playing out flaws ICly without needing a XP or AP bump for it. I don't have any alternatives to propose, save for turning 'flaws' into 'quirks' which can be naturally made to be rolled for by a GM or PrP GM as the situation calls for and likely awarded XP on the spot, maybe no more than once per week, like the random RP commands. 

Otherwise, you're looking at hundreds of characters having to get oversight for taking up flaws and that adds more overhead for staff that could go into storytelling opportunities and exciting PrP code.
